libbpf-tools/execsnoop.bpf.c

```diff
@@ -108,8 +111,14 @@ int tracepoint__syscalls__sys_enter_execve(struct trace_event_raw_sys_enter* ctx
 	return 0;
 }
 
+
+#ifdef __TARGET_ARCH_arm64
+SEC("kretprobe/do_execveat_common.isra.0")
```

Comment:
Can we handle this difference from user space ? Symbols with `isra` suffix is quite unreliable.

Reply:
Two things.
First, we should not put 'do_execveat_common.isra.0' into the section names. The function is not reliable. It depends on compiler optimization. It may be gone in the next kernel release. User space needs to handle this
to ensure this function exists before attaching to it.
Second, this is a hack for buggy kernel and these buggy LTS kernels have been fixed. So there is really no need to put such a workaround to the tool. We don't want to support any intermediate revisions of the stable kernel.